src/ansys/aedt/core/modeler/circuits/primitives_maxwell_circuit.py

```diff
@@ -284,3 +284,87 @@ def create_winding(self, name=None, location=None, angle=0, use_instance_id_netl
         )
         id.set_property("Name", name)
         return id
+
+    @pyaedt_function_handler(compname="name")
+    def create_ISin(self, name=None, value=1, location=None, angle=0, use_instance_id_netlist=False):
```

Comment:
Please avoid using capital letters in methods name, we follow the snake case convention.